scripts/pi-hole/js/ip-address-sorting.js

```diff
@@ -0,0 +1,110 @@
+/* Pi-hole: A black hole for Internet advertisements
+*  (c) 2019 Pi-hole, LLC (https://pi-hole.net)
+*  Network-wide ad blocking via your own hardware.
+*
+*  This file is copyright under the latest version of the EUPL.
+*  Please see LICENSE file for your rights under this license.  */
+jQuery.extend( jQuery.fn.dataTableExt.oSort,
+{
+    "ip-address-pre": function ( a )
+    {
+        if (!a) { return 0 }
+        var i, item;
+        var m = a.split("."),
+            n = a.split(":"),
+            x = "",
+            xa = "";
```

Comment:
Can we avoid the very short variable names, for readability?

Reply:
I straightaway copied this from the link mentioned in the commit https://datatables.net/plug-ins/sorting/ip-address description. Do we really want to change this?

Reply:
In that case, can you add that link in a comment as being the source of the code?